test/cpp/jit/test_gpu.cpp

```diff
@@ -3101,37 +3125,129 @@ void testGPU_FusionSimpleBCast() {
     tv4->merge(0);
     tv4->merge(0);
 
-    // tv4->axis(0)->parallelize(ParallelType::Unroll);
+    tv0->computeAt(tv4, -1);
+    tv1->computeAt(tv4, -1);
 
-    // fusion.printMath();
-    // fusion.printKernel();
+    auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
 
-    // tv0->computeAt(tv4, -1);
-    // tv1->computeAt(tv4, -1);
+    at::Tensor t0 = at::randn({m, k}, options);
+    at::Tensor t1 = at::randn({k, n}, options);
 
-    // tv4->axis(0)->parallelize(ParallelType::BIDx);
+    at::Tensor cg_output = at::empty({m, k, n}, options);
 
-    constexpr int x = 63, y = 33, z = 15;
+    torch::jit::fuser::cuda::FusionExecutor fe;
+    fe.compileFusion(&fusion);
+    fe.runFusion({t0, t1}, {cg_output});
 
-    auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA,
-    0);
+    auto t2 = t0.unsqueeze(-1).expand({m, k, n});
+    auto t3 = t1.expand({m, k, n});
+    auto t4 = t2.add(t3);
 
-    at::Tensor t0 = at::randn({x, y}, options);
-    at::Tensor t1 = at::randn({y, z}, options);
+    TORCH_CHECK(t4.allclose(cg_output));
+  }
+}
 
-    at::Tensor cg_output = at::empty({x, y, z}, options);
+void testGPU_FusionComplexBCast() {
+  {
```

Comment:
can we split this into separate test functions?

Reply:
Sure, would you please push a commit for it to this pr?